smart_campus/app/templates/app/question_edit_page.html

```diff
@@ -0,0 +1,106 @@
+{% extends 'app/base.html' %}
+{% load static %}
+
+{% block stylesheet %}
+  {{ block.super }}
+  <link rel="stylesheet" type="text/css" href="{% static "app/css/edit.css" %}">
+{% endblock %}
+
+{% block content %}
+<div class="ui text container">
+  <form class="ui big borderless form" action="" method="POST" id="question_form">
+    {% csrf_token %}
+    <div class="required five wide field">
+      <label>題目內容</label>
+      <input type="text" name="content" value="{{ form_data.content }}">
+    </div>
+    <div class="grouped fields">
+        <label>選項及答案勾選</label>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            {% if form_data.answer == 'choice1' %}
+              <input type="radio" name="answer" value="choice1" checked="cheked">
+            {% else %}
+              <input type="radio" name="answer" value="choice1">
+            {% endif %}
+            <label>選項一</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice1" value="{{ form_data.choice1 }}">
+        </div>
+      </div>
+      <div class="inline required fields">
+          <div class="field">
+            <div class="ui toggle checkbox">
+              {% if form_data.answer == 'choice2' %}
+                <input type="radio" name="answer" value="choice2" checked="checked">
+              {% else %}
+                <input type="radio" name="answer" value="choice1">
+              {% endif %}
+              <label>選項二</label>
+            </div>
+          </div>
+          <div class="required five wide field">
+            <input type="text" name="choice2" value="{{ form_data.choice2 }}">
+          </div>
+        </div>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            {% if form_data.answer == 'choice3' %}
+              <input type="radio" name="answer" value="choice3" checked="checked">
+            {% else %}
+              <input type="radio" name="answer" value="choice3">
+            {% endif %}
+            <label>選項三</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice3" value="{{ form_data.choice3 }}">
+        </div>
+      </div>
+      <div class="inline required fields">
+        <div class="field">
+          <div class="ui toggle checkbox">
+            {% if form_data.answer == 'choice4' %}
+              <input type="radio" name="answer" value="choice4" checked="checked">
+            {% else %}
+              <input type="radio" name="answer" value="choice4">
+            {% endif %}
+            <label>選項四</label>
+          </div>
+        </div>
+        <div class="required five wide field">
+          <input type="text" name="choice4" value="{{ form_data.choice4 }}">
+        </div>
```

Comment:
Too many duplicate codes from line 19~line77.
Refactor it.

Reply:
Refactored all code block related this.